packages/opentelemetry-metrics/src/BoundInstrument.ts

```diff
@@ -53,18 +71,16 @@ export class BoundCounter extends BaseBoundInstrument
   implements types.BoundCounter {
   constructor(
     labelSet: types.LabelSet,
-    private readonly _disabled: boolean,
-    private readonly _monotonic: boolean,
-    private readonly _valueType: types.ValueType,
-    private readonly _logger: types.Logger,
-    private readonly _onUpdate: Function
+    disabled: boolean,
+    monotonic: boolean,
+    valueType: types.ValueType,
+    logger: types.Logger,
+    aggregator: Aggregator
```

Comment:
Shouldn't the counter always use the sum aggregator?

Reply:
I know we talked about this in the SIG meeting a couple weeks ago, but I don't remember the resolution.

Reply:
As discussed offline, hardcoding will certainly simplify the things and will be less buggy (and easy to read). Also, you're right (99.9%) that `counter` should always have `CounterSumAggregator`. But I haven't seen any hardcoded stuff in specs or Go/Python implementations. Aggregators are always coming from Batcher(`aggregatorFor` - function). I was thinking to keep it as it is and open an issue to consider doing it in the future, once specs is completely flushed out.